### PR TITLE
correct python version comparison

### PR DESCRIPTION
**Pull request by lilaboc**
fix issue #245

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,10 +17,11 @@
     )
 
 import platform
+from packaging import version
```
Comment (Samyak2) [inline setup.py:20]: I couldn't find a source for this. Is it from [this package](https://pypi.org/project/packaging/)? If so, anyone wanting to install browser-history from source would need to have `packaging` installed?

**Review by Samyak2 — COMMENTED** [review]
I left a comment below.

Other than that, I think the best way to go about this would be to switch completely to a `pyproject.toml` and remove this `setup.py` altogether. @ObaraEmmanuel what do you think?

File: setup.py (hunk 1 above)
Comment (lilaboc) [reply setup.py:20]: @Samyak2  https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python
Note: packaging has recently been [vendored into setuptools](https://github.com/pypa/setuptools/tree/a94ccbf404a79d56f9b171024dee361de9a948da/pkg_resources/_vendor/packaging).

So I guess everyone should have this be default?

Comment (ObaraEmmanuel) [reply setup.py:20]: @Samyak2 Yes, I think we can explore using `pyproject.toml`

**Comment by Samyak2** [comment]
Sorry I didn't follow-up here. We have now moved to a pyproject.toml file - https://github.com/browser-history/browser-history/pull/279

**Pull request closed without merging**